js/features/scry-mode.js

```
@@ -0,0 +1,192 @@
+  try {
+    const cardName = card.isBasicByDeck ? card.displayName : card.name;
+    const stripes = card.stripes || [];
+    const el = await buildCardWithStripes(cardName, stripes);
+
+    if (scryIndex !== capturedIndex) return;
+
+    scryContent.innerHTML = '';
+    const { outer, inner } = makeScaledWrapper(scale);
+    inner.appendChild(el);
+    scryContent.appendChild(outer);
+  } catch (_err) {
```

ESLint

[error] '_err' is defined but never used.

(no-unused-vars)